src/util/analyze_util.ts

```diff
@@ -5,13 +5,17 @@ import Granularity from "../types/granularity";
 
 const MAX_ALLOWED_DATA_POINTS = 1000;
 
-export function createTagFilter(filter: TagFilter) {
-  const tagFilter = {
+export function createTagFilter(filter: TagFilter, useEntity = false) {
```

Comment:
where do we call this with true?

Reply:
We call it with true for "analyze applications". "Analyze websites" does not have the new entity addition

Reply:
My bad! That as actually an error. I fixed it with my latest commit!